gui/screens/settings.py

```diff
@@ -24,27 +28,37 @@ class LeftPanel(QObject):
 
     def __init__(self):
         super().__init__('mainWindow_LeftTabView')
-        self._settings_section_template = QObject('scrollView_AppMenuItem_StatusNavigationListItem')
+        self._settings_section_template = QObject('scrollView_MenuItem_StatusNavigationListItem')
 
-    def _open_settings(self, index: int):
-        self._settings_section_template.real_name['objectName'] = f'{index}-AppMenuItem'
+    def _open_settings(self, index: int, section_name: str):
+        self._settings_section_template.real_name['objectName'] = f'{index}-{section_name}MenuItem'
         self._settings_section_template.click()
 
     @allure.step('Open messaging settings')
     def open_messaging_settings(self) -> 'MessagingSettingsView':
-        self._open_settings(3)
+        self._open_settings(3, 'App')
```

Comment:
Can we have the situation with the same indexes but different sections? For now, I see that all indexes unique.

Reply:
Yes, we do. There are not only different indexes in real names for settings items, but also names for items, like AppMenuItem, MainMenuitem and several more